nfts/backend/server.js

```diff
@@ -0,0 +1,83 @@
+import express from "express";
+import fetch from "node-fetch";
+import bodyParser from "body-parser";
+import cors from "cors";
+import env from "dotenv";
+import { checkAPIKey } from "../../helpers/utils.mjs";
+
+const app = express();
+
+env.config();
+
+const port = 3001;
+
+app.use(
+  bodyParser.urlencoded({
+    extended: true,
+  }),
+  cors({
+    origin: "*",
+    methods: "GET,HEAD,PUT,PATCH,POST,DELETE",
+    preflightContinue: false,
+    optionsSuccessStatus: 204,
+  })
+);
+
+app.use(bodyParser.json());
+
+app.get("/", (_, res) => {
+  res.send("MetaKeep Tutorial Mock Server!");
+});
+
+app.post("/getConsentToken", async (req, res) => {
+  checkAPIKey();
```

Comment:
this check should happen when the server starts

Reply:
done